src/transactions/test/SorobanTxTestUtils.cpp

```diff
@@ -1209,26 +1210,26 @@ ContractStorageInvocationTest::putWithFootprint(
     resourceFee += refundableFee;
     auto tx = createInvokeTx(resources, makeSymbol(funcStr),
                              {keySymbol, valU64}, 1'000, resourceFee);
-    invokeTx(tx, /*expectSuccess=*/expectSuccess, false);
+    invokeTx(tx, result == INVOKE_HOST_FUNCTION_SUCCESS, false);
```

Comment:
We should check the result in `putWithFootprint` like we do for other functions.

Reply:
Good catch. Fixed